.github/workflows/docker-build.yml

```diff
@@ -0,0 +1,104 @@
+          password: ${{ secrets.DOCKERHUB_TOKEN }}
+
+  test:
+    name: Container Tests
+    runs-on: ubuntu-latest
+    if: always()
+    needs: docker-login
+    steps:
+    - name: echo-default-env-variables
+      run: |
+        echo "Home: ${HOME}"
+        echo "GITHUB_WORKFLOW: ${GITHUB_WORKFLOW}"
+        echo "GITHUB_ACTIONS: ${GITHUB_ACTIONS}"
+        echo "GITHUB_ACTOR: ${GITHUB_ACTOR}"
+        echo "GITHUB_REPOSITORY: ${GITHUB_REPOSITORY}"
+        echo "GITHUB_EVENT_NAME: ${GITHUB_EVENT_NAME}"
+        echo "GITHUB_WORKSPACE: ${GITHUB_WORKSPACE}"
+        echo "GITHUB_SHA: ${GITHUB_SHA}"
+        echo "GITHUB_REF: ${GITHUB_REF}"
+
+    - name: Check out working repository
+      uses: actions/checkout@v2
+
+    - name: Run Container Tests
+      run: |
+        curl -LO https://storage.googleapis.com/container-structure-test/latest/container-structure-test-linux-amd64
+        chmod +x container-structure-test-linux-amd64
+        sudo mv container-structure-test-linux-amd64 /usr/local/bin/container-structure-test
+        make test
+        
+  scan:
+    name: Scan for CVEs
+    runs-on: ubuntu-latest
+    if: always()
+    needs: docker-login
+    steps:
+    - name: Check out working repository
+      uses: actions/checkout@v2
+      
+    - name: Build Image
+      run: make image
+
+    - name: Scan image
+      id: scan
+      uses: anchore/scan-action@v2
+      with:
+        image: "pegasystems/pega-ready:latest"
+        fail-build: true
+        severity-cutoff: "high"
+        acs-report-enable: true
+        
+    - name: Log Scan Results
+      if: always()
+      run: |
+        echo "Failures:"
+        cat ${{ steps.scan.outputs.sarif }} | jq '.runs[0].tool.driver.rules[].shortDescription.text' | grep -i "critical" || true
+        cat ${{ steps.scan.outputs.sarif }} | jq '.runs[0].tool.driver.rules[].shortDescription.text' | grep -i "high" || true
+        echo "Warnings:"
+        cat ${{ steps.scan.outputs.sarif }} | jq '.runs[0].tool.driver.rules[].shortDescription.text' | grep -i "medium" || true
+        cat ${{ steps.scan.outputs.sarif }} | jq '.runs[0].tool.driver.rules[].shortDescription.text' | grep -i "low" || true
+        cat ${{ steps.scan.outputs.sarif }} | jq '.runs[0].tool.driver.rules[].shortDescription.text' | grep -iv "critical\|high\|medium\|low" || true
+      
+    - name: Publish Scan Results as Artifact
+      if: always()
+      uses: actions/upload-artifact@v2
+      with:
+        name: docker-scan-results
+        path: ${{ steps.scan.outputs.sarif }}
+        
+    - name: Upload Anchore Scan SARIF Report
+      uses: github/codeql-action/upload-sarif@v1
+      with:
+        sarif_file: ${{ steps.scan.outputs.sarif }}
+
+  push:
+    name: Push to DockerHub
+    if: ${{ github.ref == 'refs/heads/main' && github.repository == 'pegasystems/docker-pega-web-ready' }}
+    runs-on: ubuntu-latest
+    needs: [scan, test]
+    steps:
+      - run: make push
```

Comment:
I tried Github actions workflow on my forked repository [here](https://github.com/kishorv10/docker-pega-web-ready/blob/master/.github/workflows/docker-build.yml). This stage needs more steps, code checkout and docker login. Below sets of actions worked for me.
```
steps:
      - name: Login to Docker Hub
        uses: docker/login-action@v1
        with:
          username: ${{ secrets.DOCKERHUB_USERNAME }}
          password: ${{ secrets.DOCKERHUB_TOKEN }}
      - name: Check out working repository
        uses: actions/checkout@v2
      - name: Push image to dockerhub 
        run: make push
 ```

Reply:
updated